pruning/manager_test.go

```diff
@@ -0,0 +1,274 @@
+package pruning_test
+
+import (
+	"container/list"
+	"fmt"
+
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/pruning"
+	"github.com/cosmos/cosmos-sdk/pruning/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/tendermint/libs/log"
+	db "github.com/tendermint/tm-db"
+)
+
+func Test_NewManager(t *testing.T) {
```

Comment:
Please skip the underscores, you want to have test names show up, so TestNewManager. Use underscores after you have the name prefix up firstly and are maybe testing a section of NewManager.